Controller/Adminhtml/Events/Retry.php

```diff
@@ -15,39 +15,28 @@
 class Retry extends Action implements HttpPostActionInterface
 {
     /**
-     * @var PageFactory
-     */
-    protected $resultPageFactory;
-
-    /**
-     * @var RetryManager
-     */
-    private $retryManager;
-
-    /**
-     * @var SerializerInterface
-     */
-    private $serializer;
-
-    /**
+     * Retry Constructor
+     *
      * @param Context $context
      * @param PageFactory $resultPageFactory
      * @param RetryManager $retryManager
      * @param SerializerInterface $serializer
      */
     public function __construct(
         Context $context,
-        PageFactory $resultPageFactory,
-        RetryManager $retryManager,
-        SerializerInterface $serializer
+        private readonly PageFactory $resultPageFactory,
+        private readonly RetryManager $retryManager,
+        private readonly SerializerInterface $serializer
     ) {
         parent::__construct($context);
-        $this->resultPageFactory = $resultPageFactory;
-        $this->retryManager = $retryManager;
-        $this->serializer = $serializer;
     }
 
-    public function execute()
+    /**
+     * Execute page load
+     *
+     * @return void
+     */
+    public function execute(): void
     {
         $data = $this->getRequest()->getPostValue()['general'];
```

Comment:
I'm assuming this is because of an extra configuration node in ui component xml file. There should be a way remove the need for `['general']`. I will take a look into this in a following patch version